content/en/tracing/trace_collection/custom_instrumentation/android/otel.md

```diff
@@ -0,0 +1,482 @@
+   The tracking consent can be one of the following values [See Tracking Consent][6].:
+   * `TrackingConsent.PENDING`: The SDK starts collecting and batching the data but does not send it to the data
+     collection endpoint. The SDK waits for the new tracking consent value to decide what to do with the batched data.
+   * `TrackingConsent.GRANTED`: The SDK starts collecting the data and sends it to the data collection endpoint.
+   * `TrackingConsent.NOT_GRANTED`: The SDK does not collect any data. You will not be able to manually send any logs, traces, or
+     RUM events.
+
+   To update the tracking consent after the SDK is initialized, call: `Datadog.setTrackingConsent(<NEW CONSENT>)`.
+   The SDK changes its behavior according to the new consent. For example, if the current tracking consent is `TrackingConsent.PENDING` and you update it to:
+   * `TrackingConsent.GRANTED`: The SDK sends all current batched data and future data directly to the data collection endpoint.
+   * `TrackingConsent.NOT_GRANTED`: The SDK wipes all batched data and does not collect any future data.
+
+   Use the utility method `isInitialized` to check if the SDK is properly initialized:
+
+   ```kotlin
+    if (Datadog.isInitialized()) {
+        // your code here
+    }
+   ```
+   When writing your application, you can enable development logs by calling the `setVerbosity` method. All internal messages in the library with a priority equal to or higher than the provided level are then logged to Android's Logcat:
+   ```kotlin
+   Datadog.setVerbosity(Log.INFO)
+   ```
+
+3. Configure and enable Trace feature:
+
+   {{< tabs >}}
+   {{% tab "Kotlin" %}}
+   ```kotlin
+        val traceConfig = TraceConfiguration.Builder().build()
+        Trace.enable(traceConfig)
+   ```
+   {{% /tab %}}
+
+   {{% tab "Java" %}}
+   ```java
+        final TraceConfiguration traceConfig = TraceConfiguration.Builder().build();
+        Trace.enable(traceConfig);
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}    
+
+
+4. Datadog tracer implements the [Open Telemetry standard][18]. Create the OpenTelemetryTraceProvider and register the OpenTelemetrySdk as GlobalTelemetry instance in your application's `onCreate()` method.
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+GlobalOpenTelemetry.set(object : OpenTelemetry {
+            private val tracerProvider = OtelTracerProvider.Builder()
+                .setService([BuildConfig.APPLICATION_ID])
+                .build()
+
+            override fun getTracerProvider(): TracerProvider {
+                return tracerProvider
+            }
+
+            override fun getPropagators(): ContextPropagators {
+                return ContextPropagators.noop()
+            }
+        })
+ // and later on if you want to access the tracer provider
+ val tracerProvider = GlobalOpenTelemetry.get().getTracer(instrumentationName = "<instrumentation_name>")
```

Comment:
Given the fact, we don't do anything with `instrumentationName`, let's make it obvious by passing the empty string.

```suggestion
 val tracerProvider = GlobalOpenTelemetry.get().getTracer(instrumentationName = "")
```

Reply:
yes but in this case I want to stay consistent with Otel. Maybe later one we will use it for something. Not sure here